src/OpenLoco/GameCommands/UpdateOwnerStatus.cpp

```diff
@@ -0,0 +1,25 @@
+#include "../CompanyManager.h"
+#include "GameCommands.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    // 0x004383CA
+    static uint32_t updateOwnerStatus(const uint8_t flags, OwnerStatus ownerStatus)
+    {
+        if (flags & Flags::apply)
+        {
+            auto companyId = CompanyManager::getUpdatingCompanyId();
+            auto company = CompanyManager::get(companyId);
```

Comment:
As this is a networked thing we should check for nullptr on return. If it is nullptr return FAILURE

Reply:
Like so?